src/renderer/components/AircraftSection/index.tsx

```diff
@@ -55,15 +56,16 @@ type TransferredProps = {
 type ConnectedAircraftSectionProps = {
     selectedtrack: ModTrack,
     installedtrack: ModTrack,
-    installstatus : InstallStatus,
+    installstatus: InstallStatus,
 }
 
 // All props
 type AircraftSectionProps = {
     mod: Mod,
     selectedtrack: ModTrack,
     installedtrack: ModTrack,
-    installstatus : InstallStatus,
+    installstatus : InstallStatus
+    latestVersionNames: ModAndTrackLatestVersionNames
 }
```

Comment:
Isn't `latestVersionNames` also coming from `connect()`? Should be moved to `ConnectedAircraftSectionProps`.
`AircraftSectionProps` should be an intersection type of `ConnectedAircraftSectionProps & TransferredProps` in that case.

Reply:
talked with Holland about it converting to an intersection type before, but yes to reach this, additions should only go to connected or transferred. Agree on that

Reply:
will do